examples/src/tools/google_trends.ts

```diff
@@ -0,0 +1,9 @@
+import { GoogleTrendsAPI } from "@langchain/community/tools/google_trends";
```

Comment:
Can we call this `SERPGoogleTrendsTool`? Just so it's clear this is not officially Google